sw/device/tests/BUILD

```diff
@@ -3047,14 +3056,47 @@ _LC_STATES_DEBUG_DISALLOWED = [
     CONST.LCV.PROD_END,
 ]
 
+[
+    opentitan_test(
+        name = "rv_dm_lc_disabled_tl_{}".format(lc_state),
+        srcs = ["rv_dm_lc_disabled.c"],
+        cw310 = new_cw310_params(
+            bitstream = "//hw/bitstream:rom_with_fake_keys_otp_{}".format(lc_state),
+        ),
+        exec_env = {
+            "//hw/top_earlgrey:fpga_cw310_rom_with_fake_keys": None,
+        },
+        rsa_key = rsa_key_for_lc_state(
+            RSA_ONLY_KEY_STRUCTS,
+            lc_state_val,
+        ),
+        deps = [
+            "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+            "//sw/device/lib/base:mmio",
+            "//sw/device/lib/runtime:log",
+            "//sw/device/lib/testing:lc_ctrl_testutils",
+            "//sw/device/lib/testing/test_framework:ottf_main",
+        ],
+    )
+    for lc_state, lc_state_val in _RV_DM_LC_DISABLED_TL_LC_STATES
+]
+
+test_suite(
+    name = "rv_dm_lc_disabled_tl",
+    tags = ["manual"],
+    tests = [
+        ":rv_dm_lc_disabled_tl_{}".format(lc_state)
+        for lc_state, _ in _RV_DM_LC_DISABLED_TL_LC_STATES
+    ],
+)
+
 [
     opentitan_test(
         name = "rv_dm_lc_disabled_jtag_{}".format(lc_state),
         srcs = ["example_test_from_flash.c"],
         cw310 = cw310_jtag_params(
             bitstream = "//hw/bitstream:rom_with_fake_keys_otp_{}".format(lc_state),
-            test_cmd = ("--bootstrap=\"{firmware}\"" if lc_state_val != CONST.LCV.TEST_LOCKED0 else "") +
-                       (" --expect-fail" if lc_state_val in _LC_STATES_DEBUG_DISALLOWED else ""),
+            test_cmd = " --expect-fail" if lc_state_val in _LC_STATES_DEBUG_DISALLOWED else "",
```

Comment:
You have the second commit that changes something from the first. I am guessing you might need the `--bootstrap` to make sure that the device does not reset why doing the jtag operation?